kiosk/src/BackendRequests.tsx

```diff
@@ -53,4 +53,15 @@ export const addGameToKioskAsync = async (kioskId: string | undefined, gameShare
     catch (error) {
         throw new Error("Failed to post game to the kiosk");
     }
+}
+
+export const getGameDetailsAsync = async (gameId: string) => {
+    const gameDetailsUrl = `https://makecode.com/api/${gameId}`;
```

Comment:
use the `backendEndpoint` variable here

Reply:
I can't here. `backendEndpoint` includes `/kiosk`, and the request will fail. I can add another variable, though, with just `/api` for better readability.